internal/runners/prepare/prepare_linux.go

```diff
@@ -0,0 +1,74 @@
+package prepare
+
+import (
+	"path/filepath"
+
+	"github.com/ActiveState/cli/cmd/state-tray/pkg/autostart"
+	"github.com/ActiveState/cli/internal/appinfo"
+	"github.com/ActiveState/cli/internal/constants"
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/osutils/shortcut"
+	"github.com/ActiveState/cli/internal/rtutils"
+	"github.com/gobuffalo/packr"
+	"github.com/mitchellh/go-homedir"
+)
+
+func (r *Prepare) prepareOS() error {
+	if rtutils.BuiltViaCI { // disabled while we're still testing this functionality
+		return nil
+	}
```

Comment:
I think we can drop this now?

Reply:
Sure, should it also be removed from the equivalent Windows logic?

Reply:
Yeah let's remove it.